src/did.ts

```diff
@@ -315,7 +315,8 @@ export class DID {
       signerDid === options.capability.p.aud
     ) {
       Cacao.verify(options.capability, {
-        atTime: options.atTime,
+        // CACAO requires `atTime` in seconds
```

Comment:
Seems like we should probably be making the same change to the Cacao library?

Reply:
yeah, maybe I should block this PR until I make the same change in CACAO

Reply:
resolved. merging.